cmake/x86_64-w64-mingw32-toolchain.cmake

```diff
@@ -0,0 +1,21 @@
+cmake_minimum_required(VERSION 3.16)
```

Comment:
I'd rather this be part of [project_options](https://github.com/aminya/project_options/)